solr-updater/src/main/java/dk/dbc/search/solrdocstore/updater/BusinessLogic.java

```diff
@@ -255,19 +280,81 @@ public void attachedResources(JsonNode sourceDoc) {
         }
     }
 
+    /**
+     * Set field is any is set to true
```

Comment:
=> Set field if

Måske: Set a given field to the value true, if any of the values passed are true.